Vagrantfile

```diff
@@ -347,6 +347,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         end
         
         config.ssh.insert_key = false
+
+        $enablePswSSH = <<-SCRIPT
```

Comment:
Can you use a less cryptic name for this variable? Such as: `$enableSshPasswordAuthentication`